base/show.jl

```diff
@@ -994,7 +994,7 @@ end
 # * Print valid identifiers & operators literally; also macros names if allow_macroname=true
 # * Escape invalid identifiers with var"" syntax
 function show_sym(io::IO, sym; allow_macroname=false)
-    if isidentifier(sym) || isoperator(sym)
+    if isidentifier(sym) || (isoperator(sym) && sym !== :var"'")
```

Comment:
`Symbol("'")`? Quoting the var macro is equivalent, but seems a bit convoluted.

Reply:
Yes, probably a good idea. But the new `var""` syntax is just so nice! :grin: